apps/extension/src/core/managers/config.ts

```diff
@@ -304,6 +305,8 @@ function defaultAPILabel(model: ModelID): string {
       return "Anthropic: Claude"
     case ModelID.Claude_V1_100k:
       return "Anthropic: Claude 100k"
+    case ModelID.Shap_e:
+      return "OpenRouter: 3D Model Generator"
```

Comment:
I would call it "OpenAI: Shap-E". this label shows up in dropdowns but doesn't matter in our case because we aren't letting users select it manually yet

Reply:
agreed, changed!